src/test/java/de/retest/recheck/ignore/SearchFilterFilesTest.java

```diff
@@ -62,4 +67,20 @@ void getProjectFilterFiles_should_only_get_filter_files() throws IOException {
 		assertThat( projectFilterFiles.stream().map( Path::getFileName ) ).contains( colorFilter.toPath().getFileName(),
 				webFontFilter.toPath().getFileName() );
 	}
+
+	@Test
+	void searchFilterByName_should_return_filter_file() throws IOException {
+		final String name = "positioning.filter";
+		final String invalidName = "color.filter";
```

Comment:
I would split that into two tests ... this one and searchFilterByName_should_not_return_nonexistent_file mit color.filter.